controllers/modals/submissionMetadata/IssueEntryHandler.inc.php

```diff
@@ -60,6 +60,10 @@ function fetch($args, $request) {
 
 		$tabContentUrl = $dispatcher->url($request, ROUTE_COMPONENT, null, 'tab.issueEntry.IssueEntryTabHandler', 'galleyMetadata', null, array('submissionId' => $submission->getId(), 'stageId' => $this->getStageId()));
 		$templateMgr->assign('tabContentUrl', $tabContentUrl);
+		
+		$submissionDao = Application::getSubmissionDAO();
```

Comment:
For code within OJS, feel free to use ArticleDAO directly. It's only necessary to avoid naming it for shared code in lib-pkp.